### ISSUE TITLE
[Feature] Have social share image, but do not display in blog post

### ISSUE THREAD

**Issue by bee-san** [adityatelange/hugo-PaperMod#156]
Related to #155 

Hey,

When you add a social share image, it displays in the blog post:
![image](https://user-images.githubusercontent.com/10378052/103152420-acc5e080-477f-11eb-84c1-941e66b62a25.png)

I want the image to display in social share cards as [Summary with large image](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image), but I don't want it to display in the blog posts themselves (as Lighthouse complains about speed, and I think it's ugly).

But I do want them to appear in social share cards, as it improves click rate.

# TL;DR

Have images appear in social share cards, but not as cover photos in blog posts.

## Why?
1. Lighthouse complains it slows down the website.
2. Images in social share cards make them more clickable

**Comment by adityatelange** [comment]
@bee-san test out c9fa6c7

**Comment by adityatelange** [comment]
@bee-san test c9fa6c7 , should be able to hide cover but add it in structured data and social images
